cves/kernel/CVE-2021-27363.yml

```diff
@@ -229,8 +230,8 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
-  note:
+  name: "drivers/scsi"
+  note: "deals with the trasnportation of messages and files"
```

Comment:
1. Are these multiple subsystems or a subfolder of the drivers? If it's a subfolder that formatting should be fine, just wanted to double-check.
2. Transportation is misspelled.

Reply:
this is one subsystem! That is the directory and then the file itself I believe. I can be a little specific in my note.